ehitchhiking-ui/src/app/shared/modals/profile-modal/car-info-form/car-info-form.component.ts

```diff
@@ -10,7 +9,7 @@ export class CarInfoFormComponent implements OnInit {
 	@Input() carInfoForm: FormGroup;
 	@Input() addCarMod: boolean;
 	@Input() carIndex: number;
-	@Output() onChangeCarInfo = new EventEmitter<Car>();
+	@Output() onChangeCarInfo = new EventEmitter<any>();
```

Comment:
Why any? Pls create an interface for the emitted value

Reply:
done